puppet/modules/freight/manifests/init.pp

```diff
@@ -1,13 +1,12 @@
 # @summary install freight
 class freight {
-  if $facts['os']['family'] == 'Debian' {
-    apt::source { 'freight':
-      location => 'http://build.openvpn.net/debian/freight_team',
-      repos    => 'main',
-      key      => {
-        id     => '30EBF4E73CCE63EEE124DD278E6DA8B4E158C569',
-        source => 'https://swupdate.openvpn.net/repos/repo-public.gpg',
-      },
+  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '9' {
+    yumrepo { 'evgeni_freight':
```

Comment:
Is there something to request it in EPEL 9 proper?

Reply:
I did not follow https://docs.fedoraproject.org/en-US/epel/epel-package-request/ yet to request it.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=2299182